packages/jupyterlab-chat/src/token.ts

```diff
@@ -56,7 +57,7 @@ export interface IChatFactory {
   /**
    * The chat panel tracker.
    */
-  tracker: IWidgetTracker<LabChatPanel>;
+  tracker: WidgetTracker<LabChatPanel | ChatWidget>;
```

Comment:
Do we need to expose a full `WidgetTracker`? Or could we stick to the more limited `IWidgetTracker` as before?

Reply:
+1

Reply:
Yes, this is required in the current implementation, because the side panel uses the `add` method (not exposed in the interface) when creating a widget.
```ts
factory.tracker.add(widget);
```